Package.swift

```diff
@@ -26,6 +26,7 @@ let package = Package(
                 ],
                 sources: [
                     "src/parser.c",
+                    "src/scanner.c",
```

Comment:
I don't know anything about swift, so I am not sure whether this change is correct/how to test it (but happy to have a go if others can explain).

Reply:
I'm unsure as well but I suspect this is all that's needed.

cc// @mattmassicotte let us know if there's anything else that's needed

Reply:
This looks good - thanks for keeping it up to date!